resources/views/depreciations/edit.blade.php

```diff
@@ -22,5 +22,15 @@
     </div>
     {!! $errors->first('months', '<span class="alert-msg" aria-hidden="true"><i class="fa fa-times" aria-hidden="true"></i> :message</span>') !!}
 </div>
+<!-- Min Depreciation -->
+    <label for="min-value" class="col-md-3 control-label">
+        {{ trans('admin/depreciations/general.min-value') }}
+    </label>
+        <div class="col-md-2" style="padding-left:0px">
+            <input class="form-control" type="number" step="0.01" min="0.00" name="min-value" id="min-value" value="{{ Request::old('min_value', $item->min_value) }}" style="width: 80px;" />
```

Comment:
Looks like a conflict here in the `name="min-value" ` vs `old('min_value`)

Reply:
Also I assume this is doing a number-stepper thing? (A screenshot of what this looks like on your local would be helpful) - we probably don't want a number stepper if this is for a currency - it can be a clunky UX to make people click 100 times if they want the number to be 100, etc.